app/src/main/java/com/github/se/icebreakrr/model/profile/ProfilesRepositoryFirestore.kt

```diff
@@ -177,17 +176,17 @@ class ProfilesRepositoryFirestore(
         }
   }
 
```

Comment:
You can add documentation here to explain what this function does, where and why it is called,

Reply:
Yes I can add that !